### ISSUE TITLE
Bug report when enable Nginx after update 24.7

### ISSUE THREAD

**Issue by netcedec** [opnsense/plugins#4113]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [x ] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [x ] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [x ] The title contains the plugin to which this issue belongs

**Describe the bug**
After upgrading to 24.7 a bug report appeared, nginx and waf naxsi logs are missing. When restarting the service, the bug report appears. Nginx itself proxies, but no more.
The nginx and waf configurations have not been changed since the upgrade.

**To Reproduce**
1. Go to opnsense
2. Click on update
3. After update go in Loby -> Dashboard
4. Add Widget "Services"
5. Save widget
6. Click on reload service "Reverse Proxy and Web Server"
7.  Go Services -> Nginx -> Logs / HTTP Access or HTTP Error

**Expected behavior**
Service reload, there are logs in Nginx

**Screenshots**

![image](https://github.com/user-attachments/assets/99185326-d7ff-48f7-9dc4-79a6851418ee)

![image](https://github.com/user-attachments/assets/f9d66e90-6e31-4537-9373-fb9bff807bf7)

**Relevant log files**
In bug report 

PHP Error:

```
[25-Jul-2024 18:08:57 Europe/Moscow] Exception: Response Already Sent in /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Response.php:97
Stack trace:
#0 /usr/local/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php(356): OPNsense\Mvc\Response->send()
#1 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Dispatcher.php(167): OPNsense\Base\ApiControllerBase->afterExecuteRoute(Object(OPNsense\Mvc\Dispatcher))
#2 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(156): OPNsense\Mvc\Dispatcher->dispatch(Object(OPNsense\Mvc\Request), Object(OPNsense\Mvc\Response), Object(OPNsense\Mvc\Session))
#3 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(139): OPNsense\Mvc\Router->performRequest(Object(OPNsense\Mvc\Dispatcher))
#4 /usr/local/opnsense/www/api.php(36): OPNsense\Mvc\Router->routeRequest('/api/nginx/logs...', Array)
#5 {main}
[25-Jul-2024 18:08:58 Europe/Moscow] Exception: Response Already Sent in /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Response.php:97
Stack trace:
#0 /usr/local/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php(356): OPNsense\Mvc\Response->send()
#1 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Dispatcher.php(167): OPNsense\Base\ApiControllerBase->afterExecuteRoute(Object(OPNsense\Mvc\Dispatcher))
#2 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(156): OPNsense\Mvc\Dispatcher->dispatch(Object(OPNsense\Mvc\Request), Object(OPNsense\Mvc\Response), Object(OPNsense\Mvc\Session))
#3 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(139): OPNsense\Mvc\Router->performRequest(Object(OPNsense\Mvc\Dispatcher))
#4 /usr/local/opnsense/www/api.php(36): OPNsense\Mvc\Router->routeRequest('/api/nginx/logs...', Array)
#5 {main}
[25-Jul-2024 18:09:04 Europe/Moscow] Exception: Response Already Sent in /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Response.php:97
Stack trace:
#0 /usr/local/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php(356): OPNsense\Mvc\Response->send()
#1 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Dispatcher.php(167): OPNsense\Base\ApiControllerBase->afterExecuteRoute(Object(OPNsense\Mvc\Dispatcher))
#2 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(156): OPNsense\Mvc\Dispatcher->dispatch(Object(OPNsense\Mvc\Request), Object(OPNsense\Mvc\Response), Object(OPNsense\Mvc\Session))
#3 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(139): OPNsense\Mvc\Router->performRequest(Object(OPNsense\Mvc\Dispatcher))
#4 /usr/local/opnsense/www/api.php(36): OPNsense\Mvc\Router->routeRequest('/api/nginx/logs...', Array)
#5 {main}
[25-Jul-2024 18:09:06 Europe/Moscow] Exception: Response Already Sent in /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Response.php:97
Stack trace:
#0 /usr/local/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php(356): OPNsense\Mvc\Response->send()
#1 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Dispatcher.php(167): OPNsense\Base\ApiControllerBase->afterExecuteRoute(Object(OPNsense\Mvc\Dispatcher))
#2 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(156): OPNsense\Mvc\Dispatcher->dispatch(Object(OPNsense\Mvc\Request), Object(OPNsense\Mvc\Response), Object(OPNsense\Mvc\Session))
#3 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(139): OPNsense\Mvc\Router->performRequest(Object(OPNsense\Mvc\Dispatcher))
#4 /usr/local/opnsense/www/api.php(36): OPNsense\Mvc\Router->routeRequest('/api/nginx/logs...', Array)
#5 {main}
[25-Jul-2024 18:09:06 Europe/Moscow] Exception: Response Already Sent in /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Response.php:97
Stack trace:
#0 /usr/local/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php(356): OPNsense\Mvc\Response->send()
#1 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Dispatcher.php(167): OPNsense\Base\ApiControllerBase->afterExecuteRoute(Object(OPNsense\Mvc\Dispatcher))
#2 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(156): OPNsense\Mvc\Dispatcher->dispatch(Object(OPNsense\Mvc\Request), Object(OPNsense\Mvc\Response), Object(OPNsense\Mvc\Session))
#3 /usr/local/opnsense/mvc/app/library/OPNsense/Mvc/Router.php(139): OPNsense\Mvc\Router->performRequest(Object(OPNsense\Mvc\Dispatcher))
#4 /usr/local/opnsense/www/api.php(36): OPNsense\Mvc\Router->routeRequest('/api/nginx/logs...', Array)
#5 {main}
```

**Additional context**

**Environment**
Software version used and hardware type if relevant.
e.g.:

OPNsense 24.7-amd64
Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz
Network VMXNET 3

**Comment by ivulit** [comment]
Same issue here after upgrade. No access/error logs and got same issue report. os-nginx version is 1.34.

**Comment by kulikov-a** [comment]
can you test with
`opnsense-patch -c plugins -a kulikov-a a7ef2d9` (https://github.com/opnsense/plugins/commit/a7ef2d9a29e858d6700a81aab22417df9a1adb0c)
please?

**Comment by ivulit** [comment]
> can you test with `opnsense-patch -c plugins -a kulikov-a a7ef2d9` ([a7ef2d9](https://github.com/opnsense/plugins/commit/a7ef2d9a29e858d6700a81aab22417df9a1adb0c)) please?

Hi. i've applied this patch. Access/Error logs are available again and no any bug reports. Looks like patch solved this issue.

**Comment by kulikov-a** [comment]
@ivulit great! thanks, i'll clean it up and make a pr a little later

**Comment by netcedec** [comment]
> can you test with `opnsense-patch -c plugins -a kulikov-a a7ef2d9` ([a7ef2d9](https://github.com/opnsense/plugins/commit/a7ef2d9a29e858d6700a81aab22417df9a1adb0c)) please?

Thanks! It's work!!!!!!!!!

**Comment by ivulit** [comment]
@kulikov-a Thanks a lot for quick response!

**Comment by GitTimeraider** [comment]
Can also confirm the update command fixes it. Hopefully it gets through into the next update so everyone can have it forced through :P

**Comment by OoSTARTERoO** [comment]
quick and easy fix! thanks!

**Comment by fichtner** [comment]
It has been merged now but I missed this for 24.7.1. Sorry.
